pkg/proxy/integrations/postgresParser/utils.go

```diff
@@ -294,7 +276,6 @@ func ChangeAuthToMD5(tcsMocks []*models.Mock, h *hooks.Hook, log *zap.Logger) {
 
 				log.Debug("CHANGING TO MD5 for Response")
 				mock.Spec.GenericResponses[requestIndex].Message[0].Data = "UgAAAAwAAAAF4I8BHg=="
-				// isScram = true
```

Comment:
don't remove this comment, as it is for debugging purpose.

Reply:
Only // isScram = true right?

Reply:
Any idea why there is issue with DCO? I have signed it.